crates/bevy_render/src/settings.rs

```diff
@@ -59,12 +59,15 @@ impl Default for WgpuSettings {
             limits
         };
 
+        let features = wgpu::Features::TEXTURE_ADAPTER_SPECIFIC_FORMAT_FEATURES;
+        #[cfg(feature = "gpu_profiler")]
+        let features = features | wgpu::Features::TIMESTAMP_QUERY;
```

Comment:
I think this will fail on platforms that don't support timestamp queries. I don't know if I think that should be the preferred behaviour rather than silently not working.